pkg/apis/config/store_test.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/tektoncd/triggers/pkg/apis/config"
-	test "github.com/tektoncd/triggers/pkg/reconciler/testing"
+	"github.com/tektoncd/triggers/test"
 	logtesting "knative.dev/pkg/logging/testing"
```

Comment:
can we add testcases here for `feature-config`

Reply:
added a test for FromContextOrDefaults which tests both config-defaults and config-features